fs/fs.go

```diff
@@ -357,6 +370,10 @@ func dockerDMDevice(driverStatus string) (string, uint, uint, uint, error) {
 	if len(poolName) == 0 {
 		return "", 0, 0, 0, fmt.Errorf("Could not get dm pool name")
 	}
+	dataLoopFile := dockerStatusValue(config, "Data loop file")
```

Comment:
I can't think of another option - the fact these keys/values are free text is a bit loose for me, but no other option.